app/models/warehouse_report/overlapping_coc_by_project_type.rb

```diff
@@ -249,7 +249,7 @@ def limited_details_clients
     services.group_by { |s| s.service_history_enrollment.project }.map do |project, project_services|
       {
         coc: project.project_cocs.first.CoCCode,
-        project_name: project.name,
+        project_name: project.name(ignore_confidential_status: true),
```

Comment:
I _think_ we want to ignore confidential projects entirely in this report.  There is a drill-down that might expose too much.

Reply:
Okay. I'll tackle that in the next PR if that's alright